setup.cfg

```diff
@@ -1,2 +1,11 @@
 [bdist_wheel]
 universal = 1
+
+[bumpversion]
+current_version = 0.1.5
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
```

Comment:
And move this bit to .bumpversion.cfg.

From past experience - keeping this in bumpversion.cfg is the easiest, as bumpversion can sometimes be a bit noisy with commits - reformatting things.